### PR TITLE
Dasc 976

### PR DESCRIPTION
**Pull request by darolt**
#### Please provide details about this Pull Request (why the change is being made, what this commit will do):
This commit was originated by dasc-976 and dasc-977. However its scope was enlarged by:
- adding more unit tests
- using linters and formatters to standardize dev.

### CODE REVIEW

File: pycarol/exceptions.py [hunk 1]
```diff
@@ -1,16 +1,31 @@
...
-        Exception (Exception): SuperClass
-    """
+    """Custom Exception to handle Invalid token."""
+
     pass
 
+
+class MissingInfoCarolException(ValueError):
+
+    """When some information is missing to Carol.__init__."""
```
Comment (fnnakamura) [inline pycarol/exceptions.py:17]: Apenas sugestão a título de padronização de docstring:
```
"""Custom exception to handle Carol missing information on Carol.__init__."""
```

File: pycarol/exceptions.py [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+
+    pass
+
+
+class DeprecatedEnvVarException(Exception):
+
+    """When some environment variable is deprecated."""
```
Comment (fnnakamura) [inline pycarol/exceptions.py:24]: Apenas sugestão a título de padronização de docstring:
```
"""
Custom exception to handle deprecated environment variables
Args:
    old_var (str): Deprecated environment variable
    new_var (str): Environment variable to be replaced
"""
```

File: pycarol/carol.py [hunk 3]
```diff
@@ -1,515 +1,470 @@
...
+            "v2/apiKey/details", params={"apiKey": api_key, "connectorId": connector_id}
+        )
 
         return resp
 
-    def api_key_revoke(self, connector_id):
-        """
-        Revoke API key for ta given connector_id
+    def api_key_revoke(self, connector_id: str) -> ResponseType:
+        """Revoke API key for ta given connector_id.
```
Comment (fnnakamura) [inline pycarol/carol.py:389]: "Revoke API key for ta? given connector_id."

"ta" é um typo pra "a" ou uma abreviação pra "tenant"?

**Review by fnnakamura — APPROVED** [review]
Great!!!

File: pycarol/exceptions.py (hunk 1 above)
Comment (darolt) [reply pycarol/exceptions.py]: corrigido: https://github.com/totvslabs/pyCarol/commit/8161f712af112ff74c04137664d9ac4dc5bc7d05

File: pycarol/exceptions.py (hunk 2 above)
Comment (darolt) [reply pycarol/exceptions.py]: corrigido: https://github.com/totvslabs/pyCarol/commit/8161f712af112ff74c04137664d9ac4dc5bc7d05

File: pycarol/carol.py (hunk 3 above)
Comment (darolt) [reply pycarol/carol.py]: corrigido: https://github.com/totvslabs/pyCarol/commit/8161f712af112ff74c04137664d9ac4dc5bc7d05

**Pull request merged**